src/acquisition/covidcast/csv_importer.py

```diff
@@ -251,33 +299,40 @@ def extract_and_check_row(row, geo_type):
     else:
       return (None, 'geo_type')
 
-    # required float
-    try:
-      value = float(row.val)
-      if math.isnan(value) or math.isinf(value):
-        raise ValueError('nan/inf not valid for `value`')
-    except (TypeError, ValueError) as e:
-      # val was either `None` or not a float
+    # Validate row values
+    value = CsvImporter.validate_value(row, "val")
+    # val was a string or another dtype
+    if value == "Error":
       return (None, 'val')
-
-    # optional nonnegative float
-    try:
-      stderr = CsvImporter.maybe_apply(float, row.se)
-    except ValueError:
-      # expected a number, but got a string
+    stderr = CsvImporter.validate_value(row, "se")
+    # stderr was a string or another dtype
+    if stderr == "Error":
       return (None, 'se')
     if stderr is not None and stderr < 0:
```

Comment:
could replace `if` with an `or` but im really picking nits now...

Reply:
Combined these and made comments to make clear what cases are being captured in each.